README.md

```diff
@@ -86,6 +86,26 @@ def register_pandora_vocab(emitter):
         for match in m.groups():
             register_vocab('Pandora Station', match)
 ```
+
+Development
+=======
+
+Glad you'd like to help!
+
+To install test and development requirements run
+
+```
+pip install -r test-requirements.txt
+```
+
+This will install the test-requirements as well as the runtime requirements for adapt.
+
+To test any changes before submitting them run
+
+```
+pytest test/*
```

Comment:
I think that we should include the flake8 command in the instructions (as that will also fail the remote build), and we should pull both into a single script for simpler instructions.